Packs/qualys/Integrations/Qualysv2/Qualysv2.yml

```
@@ -2440,13 +3180,18 @@ script:
     - contextPath: Qualys.TimeZone.TIME_ZONE_DETAILS
       description: Timezone code details.
       type: String
-    description: Gets a list of the supported time zone codes.
-    execution: true
-  dockerimage: demisto/python3:3.10.4.29342
+  - arguments:
+    - description: Record IDs to update
+      name: ids
+    - description: IPs specified will overwrite existing IPs in the record, and existing
+        IPs will be removed
+      name: add_ips
```

Since the arguments are `required`, please change to:

```suggestion
    - description: Record IDs to update
      name: ids
      required: true
    - description: IPs specified will overwrite existing IPs in the record, and existing
        IPs will be removed
      name: add_ips
      required: true
```

---

Hi, I made the changes in the following commit: https://github.com/demisto/content/pull/20142/commits/9606eb3af1a5d3a4d3c4acaa2b7012eb66d72aa4#diff-1f83dd646fa0132acc13048ae6445bf62056f3b1cbaf064a0c82679e046453e6